src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/EvalResultDataItem.cs

```diff
@@ -193,7 +199,13 @@ internal EvalResultDataItem ToDataItem()
                 FormatSpecifiers);
         }
 
-        private static DkmEvaluationResultFlags GetFlags(DkmClrValue value, DkmInspectionContext inspectionContext)
+        internal void DisableCanAddFavorite()
```

Comment:
>DisableCanAddFavorite [](start = 22, length = 21)

We usually prefer immutable object. The pattern is to create a new object using WithDisableCanAddFavorite()